gremlin-server/src/test/scripts/test-server-start.groovy

```diff
@@ -60,7 +60,10 @@ settings.graphs.modern = platformAgnosticGremlinServerDir + "/src/test/scripts/t
 settings.graphs.crew = platformAgnosticGremlinServerDir + "/src/test/scripts/tinkergraph-empty.properties"
 settings.graphs.grateful = platformAgnosticGremlinServerDir + "/src/test/scripts/tinkergraph-empty.properties"
 settings.graphs.sink = platformAgnosticGremlinServerDir + "/src/test/scripts/tinkergraph-empty.properties"
-if (testTransactions) settings.graphs.tx = platformAgnosticGremlinServerDir + "/src/test/scripts/neo4j-empty.properties"
+if (testTransactions) {
+    settings.graphs.tx = platformAgnosticGremlinServerDir + "/src/test/scripts/neo4j-empty.properties"
```

Comment:
is there any point to configuring neo4j for testing now that we have TinkerGraph? can't we just remove the configuration? iirc we only have neo4j here to validate that remote transactions work on the server and with drivers and we can now do that with TinkerGraph. we really don't need to test that neo4j itself works in this context.

Reply:
Yes, part of this work should be pulling out the Neo4j integration